build/npm/navigation-react-native/navigation.react.native.d.ts

```diff
@@ -525,6 +525,14 @@ export interface TabBarProps {
      * The default selected tab index
      */
     defaultTab?: number;
+    /**
+     * Whether to show or hide labels in the tab bar (Android Only)
+     * selected - only show on active tab
+     * auto - "show" when <4 tabs, "selected" when >3
+     * labeled - always show
+     * unlabeled - always hide (no labels)
+     */
+    labelVisibility?: "auto" | "labeled" | "unlabeled" | "selected";
```

Comment:
Thanks for writing a comprehensive typing comment. It's so good that it's inconsistent with my other comments ;). Can you change it to the following, please?
```ts
/**
 * Indicates how labels should be displayed in the tab bar
 */
labelVisibility?: "auto" | "labeled" | "unlabeled" | "selected";
```

Reply:
Sorry, did that out of habit. But you're right, I should aim to keep it consistent! Fixed this